src/flb_log.c

```diff
@@ -745,6 +750,17 @@ int flb_errno_print(int errnum, const char *file, int line)
     return 0;
 }
 
+int flb_WSAGetLastError_print(int errnum, const char *file, int line)
+{
+#ifdef WIN32
+    char buf[256];
+    FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS,
+                  NULL, errnum, MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
+                  buf, sizeof(buf), NULL);
+    flb_error("[%s:%i WSAGetLastError=%i] %s", file, line, errnum, buf);
+#endif
+}
```

Comment:
Please add a return value or change the return type to `void`

Reply:
Added a return value to keep it lined up with `flb_errno` in [7a1429c](https://github.com/fluent/fluent-bit/pull/10017/commits/7a1429c720b7ac8aa9d74943b44be7c2743678cf)